ocaml/tests/common/test_common.ml

```diff
@@ -635,16 +635,17 @@ let make_cluster_host ~__context ?(ref = Ref.make ()) ?(uuid = make_uuid ())
 let make_cluster_and_cluster_host ~__context ?(ref = Ref.make ())
     ?(uuid = make_uuid ()) ?(cluster_token = "") ?(pIF = Ref.null)
     ?(cluster_stack = Constants.default_smapiv3_cluster_stack)
-    ?(allowed_operations = []) ?(current_operations = [])
-    ?(pool_auto_join = true)
+    ?(cluster_stack_version = 3L) ?(allowed_operations = [])
```

Comment:
Why is it `3` here, but it's `2` in datamodel?

Reply:
The `2` in the datamodel is for backwards compatibility reasons, when we upgrade from an old xapi db which does not have the field, but has a cluster object, we can default that field to 2, because upgrading from an old xapi necessarily means corosync has version 2.

The 3, OTOH, is used as the "default" version when we _create_ a new cluster, since we want to enforce that whenever the user is creating a new cluster in XS 9. But in this case it does not really matter because it is not tested :)

I think this can be confusing so maybe I will add a clarification in the datamodel